test/e2e/storage/testsuites/volumelimits.go

```diff
@@ -326,7 +330,7 @@ func getCSINodeLimits(cs clientset.Interface, nodeName string, driverInfo *Drive
 		}
 		var csiDriver *storagev1beta1.CSINodeDriver
 		for _, c := range csiNode.Spec.Drivers {
-			if c.Name == driverInfo.Name {
+			if c.Name == driverInfo.Name || c.Name == config.GetUniqueDriverName() {
```

Comment:
Some drivers have their name updated (e.g., `csi-hostpath` to `csi-hostpath-namespace`).